openwisp_notifications/tasks.py

```diff
@@ -56,12 +57,12 @@ def delete_obsolete_objects(instance_app_label, instance_model, instance_id):
             return
 
 
-@shared_task
+@shared_task(base=OpenwispCeleryTask)
 def delete_notification(notification_id):
     Notification.objects.filter(pk=notification_id).delete()
 
 
-@shared_task
+@shared_task(base=OpenwispCeleryTask)
```

Comment:
this task could definitely take more than 10 seconds in some cases, I think the default timeouts are too strict, looking at this I would be even more conservative and increase the hard time limit to 2 minutes instead of 1 as I suggested in my other review.